licenses/Apache-2.0.lre

```diff
@@ -93,21 +98,24 @@ https://opensource.org/licenses/Apache-2.0
    Definitions.
 
       "License" shall mean the terms and conditions for use, reproduction, and
-      distribution as defined by Sections 1 through 9 of this document.
+      distribution as defined by Sections 1 through 9 of __1__ document.
 
-      "Licensor" shall mean the copyright owner or entity authorized by the
+      (("Licensor" || "Restream"))
```

Comment:
I'm not wild about hard-coding what I assume is a company name here (Restream).
Perhaps this should be just __1__. ?

Reply:
Done.